### PR TITLE
Audit innerHTML usage and sanitize inputs.

### PR DESCRIPTION
**Pull request by afshin**
## Code changes

Audits `innerHTML` usage and sanitizes strings where necessary.

## User-facing changes

N/A

### CODE REVIEW

File: packages/outputarea/src/widget.ts [hunk 1]
```diff
@@ -386,7 +386,10 @@ export class OutputArea extends Widget {
       }
       output.renderModel(model).catch(error => {
         // Manually append error message to output
-        output.node.innerHTML = `<pre>Javascript Error: ${error.message}</pre>`;
+        const pre = document.createElement('p');
```
Comment (ian-r-rose) [inline packages/outputarea/src/widget.ts]: Is this change from `<pre>` to `<p>` intended?

Comment (jasongrout) [reply packages/outputarea/src/widget.ts]: No, thanks for catching this.

**Comment by ian-r-rose** [comment]
Thanks!

**Comment by blink1073** [comment]
@meseeksdev backport to 0.35.x

**Comment by blink1073** [comment]
@meeseeksdev backport to 0.35.x

**Pull request merged**